distribution/build.gradle

```diff
@@ -285,6 +285,14 @@ configure(subprojects.findAll { ['deb', 'rpm'].contains(it.name) }) {
       we see it. We'll add it back on to the file name though. */
     version project.version.replace('-SNAPSHOT', '')
 
+    // signing setup
+    signingKeyId = project.hasProperty('signing.keyId') ? project.getProperty('signing.keyId') : System.getProperty('signing.keyId', 'D88E42B4')
```

Comment:
getProperty() is not a method on Project, it is just property(). Also, I don't think we should mess with both gradle properties and java system properties. We can just use gradle properties, passing them in like `-Psigning.password=foo`. Finally, I don't think we should set this default for signing.keyId. But we should have a default for the keyring file? See how the [nexus plugin](https://github.com/bmuschko/gradle-nexus-plugin/blob/master/src/main/groovy/com/bmuschko/gradle/nexus/NexusPlugin.groovy#L110) does it.